template/subscription_template.json

```diff
@@ -51,5 +52,11 @@
         "autoDeleteOnIdle": "P10675199DT2H48M5.4775807S"
       }
     }
-  ]
+  ],
+  "output": {
+    "resourceId": {
+      "value": "[resourceId('Microsoft.ServiceBus/namespaces/topics/subscriptions', variables('subscriptionName'))]",
```

Comment:
Does it work?

Reply:
Don't know yet, but willing to give it a go.